shotgun_api3/shotgun.py

```diff
@@ -2342,12 +2321,13 @@ def _upload_to_sg(self, entity_type, entity_id, path, field_name, display_name,
         attachment_id = int(str(result).split(":")[1].split("\n")[0])
         return attachment_id
 
-    def _get_attachment_upload_info(self, is_thumbnail, filename):
+    def _get_attachment_upload_info(self, is_thumbnail, filename, multipart_upload):
         """
         Internal function to get the information needed to upload a file to Cloud storage.
 
         :param bool is_thumbnail: indicates if the attachment is a thumbnail.
         :param str filename: name of the file that will be uploaded.
+        :param bool multipart_upload: Indicates if we want multi-part upload information back.
```

Comment:
I intentionally kept this somewhat opaque to help keep users away from the details...